pylibftdi/driver.py

```diff
@@ -72,36 +73,43 @@ class Driver(object):
     """
 
     # prefer libftdi1 if available. Windows uses 'lib' prefix.
-    _lib_search = {
-        'libftdi': ('ftdi1', 'libftdi1', 'ftdi', 'libftdi'),
-        'libusb': ('usb-1.0', 'libusb-1.0')
+    _lib_search: dict[str, list[str]] = {
+        'libftdi': ['ftdi1', 'libftdi1', 'ftdi', 'libftdi'],
+        'libusb': ['usb-1.0', 'libusb-1.0']
     }
 
-    def __init__(self, libftdi_search=None, **kwargs):
+    def __init__(self, libftdi_search: Optional[str]=None, **kwargs: dict[str, Any]) -> None:
         """
         :param libftdi_search: force a particular version of libftdi to be used
             can specify either library name(s) or path(s)
         :type libftdi_search: string or sequence of strings
         """
         if libftdi_search is not None:
-            self._lib_search['libftdi'] = libftdi_search
+            self._lib_search['libftdi'] = list(libftdi_search)
 
-    def _load_library(self, name, search_list=None):
+        # Library handles.
+        self._fdll: Any = None
+        self._libusb_dll: Any = None
+
+    def _load_library(self, name: str, search_list: Optional[str]=None) -> Any:
         """
         find and load the requested library
 
         :param name: library name
-        :param search_list: sequence or string referring to library names
+        :param search_list: string referring to library names
```

Comment:
as in previous comments, I think having Optional[List[str]] at this point satisfies more use-cases. I don't particularly mind at this point in the code, since `_load_library` is an internal function, but I think that's probably the cleanest approach.

Reply:
Agreed. I've made the appropriate changes to make the type explicitly an `Optional[list[str]]`.